src/ServerDaemon.chpl

```diff
@@ -795,6 +774,7 @@ module ServerDaemon {
             this.socket = this.context.socket(ZMQ.REP); 
             this.port = try! getEnv('METRICS_SERVER_PORT','5556'):int;
 
+            try! this.socket.bind("tcp://*:%?".format(this.port));
             try! this.socket.bind("tcp://*:%?".format(this.port));
```

Comment:
it seems we duplicated this line, i'm not sure if that was intentional or not (I'm guessing not since the port didn't change

Reply:
Yeah not intentional, must have been some weirdness from resolving merge conflicts. Thanks for catching that!